src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/Utilities.cs

```diff
@@ -144,8 +144,8 @@ internal static object GetPrimitiveDefaultValue(Type type)
                     return default(float);
                 case TypeCode.Double:
                     return default(double);
-                //case TypeCode.DBNull:
-                //    return default(DBNull);
+                case TypeCode.DBNull:
+                    return default(DBNull);
```

Comment:
I'm reordering the chain in TypeExtensions right now, which is what made me spot this. Shall I back away for now (it'll be a merge nightmare if we both hit it) or add in DBNull?